src/backend/metal/src/native.rs

```diff
@@ -204,18 +210,92 @@ pub struct ModuleInfo {
     pub rasterization_enabled: bool,
 }
 
+#[cfg(feature = "pipeline-cache")]
```

Comment:
would it help to put all of the `pipeline-cache`-releated values into a submodule? It would allow us to have a single `#[cfg(feature = "pipeline-cache")]` as well as group them together.